h/templates/activity/search.html.jinja2

```diff
@@ -2,6 +2,33 @@
 
 {% from '../includes/annotation_card.html.jinja2' import annotation_card %}
 
+{% macro search_result_nav(group_name) %}
+<nav class="search-result-nav">
+  {%- if more_info %}
+    <form method="POST"> {# This <form> is needed for IE because it doesn't
```

Comment:
We're starting to repeat this pattern in quite a few places. You might want to factor it into a macro.